years_to_hours.rb

```diff
@@ -0,0 +1,17 @@
+puts "Enter a number of years"
+years = gets.chomp # this returns a string
+years = years.to_i # this converts a string to an integer
+hours = years * 365 * 24
+puts "That's #{hours} hours."
+
+puts “Enter number of decades”
+decades= gets.chomp.to_i # this returns a string
+minutes= decades * 365 * 24*60
+puts " Thats #{minutes}minutes."
+
+puts "What is your age?"
+age = seconds.get.chomp.to_i # this return a string
+seconds = age * 365 *24*60*60
+puts "WOW! You are #{seconds} seconds old!"
+
```

Comment:
It's great to see that you turned the string into an integer with one line of code instead of 2. On Line 9, the should multiply by 10 again because its talking about decades, not years, and there is a space missing between the two minutes. Looks great